Class/C-Array-Methods/exercise5.js

```diff
@@ -1,14 +1,19 @@
 
-let namesList = ["Sundar", "Reshma", "Shantanu", "Arvind"];
 
 // 1. Write a function called capitaliseName 
 // It should have one input parameter called theName
 // It should return the name in capitals - do an internet search for javascript toUpperCase
 
 
 
-// 2. Use array.map to create a new array with all the values in namesList converted to capitals
+function capitaliseName (theName) {
+    return theName.toUpperCase();
+}
 
 
+// 2. Use array.map to create a new array with all the values in namesList converted to capitals
+let messyNames = ["Sundar", "Reshma", "Shantanu", "Arvind"];
+let theName = messyNames.map(capitaliseName);
+    console.log(theName);
 

```

Comment:
Excellent. I recommend re-naming `theName` on line 16 to something that makes it clearer what it is e.g. `capNames`. It also means that it's different from the name of the parameter in the `capitaliseName` function. 

This doesn't affect the computer but it makes it easier for a human to read and maintain the code. This becomes more and more important as code gets more complicated.